RDU-Scale/Ocata/openshift-scalelab-ci/network-environment.yaml

```diff
@@ -0,0 +1,63 @@
+  OS::TripleO::R630Compute::Ports::ExternalPort: /usr/share/openstack-tripleo-heat-templates/network/ports/noop.yaml
+  OS::TripleO::R630Compute::Ports::InternalApiPort: /usr/share/openstack-tripleo-heat-templates/network/ports/internal_api.yaml
+  OS::TripleO::R630Compute::Ports::StoragePort: /usr/share/openstack-tripleo-heat-templates/network/ports/storage.yaml
+  OS::TripleO::R630Compute::Ports::StorageMgmtPort: /usr/share/openstack-tripleo-heat-templates/network/ports/noop.yaml
+  OS::TripleO::R630Compute::Ports::TenantPort: /usr/share/openstack-tripleo-heat-templates/network/ports/tenant.yaml
+  OS::TripleO::P1029Compute::Net::SoftwareConfig: /home/stack/templates/nic-configs/1029p-compute.yaml
+  OS::TripleO::P1029Compute::Ports::ExternalPort: /usr/share/openstack-tripleo-heat-templates/network/ports/noop.yaml
+  OS::TripleO::P1029Compute::Ports::InternalApiPort: /usr/share/openstack-tripleo-heat-templates/network/ports/internal_api.yaml
+  OS::TripleO::P1029Compute::Ports::StoragePort: /usr/share/openstack-tripleo-heat-templates/network/ports/storage.yaml
+  OS::TripleO::P1029Compute::Ports::StorageMgmtPort: /usr/share/openstack-tripleo-heat-templates/network/ports/noop.yaml
+  OS::TripleO::P1029Compute::Ports::TenantPort: /usr/share/openstack-tripleo-heat-templates/network/ports/tenant.yaml
+  OS::TripleO::R720xdCompute::Net::SoftwareConfig: /home/stack/templates/nic-configs/r720xd-compute.yaml
+  OS::TripleO::R720xdCompute::Ports::ExternalPort: /usr/share/openstack-tripleo-heat-templates/network/ports/noop.yaml
+  OS::TripleO::R720xdCompute::Ports::InternalApiPort: /usr/share/openstack-tripleo-heat-templates/network/ports/internal_api.yaml
+  OS::TripleO::R720xdCompute::Ports::StoragePort: /usr/share/openstack-tripleo-heat-templates/network/ports/storage.yaml
+  OS::TripleO::R720xdCompute::Ports::StorageMgmtPort: /usr/share/openstack-tripleo-heat-templates/network/ports/noop.yaml
+  OS::TripleO::R720xdCompute::Ports::TenantPort: /usr/share/openstack-tripleo-heat-templates/network/ports/tenant.yaml
+
+
+
+parameter_defaults:
+  NeutronExternalNetworkBridge: "''"
+  InternalApiNetCidr: 172.16.0.0/16
+  TenantNetCidr: 172.17.0.0/16
+  StorageNetCidr: 172.18.0.0/16
+  StorageMgmtNetCidr: 172.19.0.0/16
+  ManagementNetCidr: 172.20.0.0/16
+  ExternalNetCidr: 172.21.0.0/16
+  InternalApiAllocationPools: [{'start': '172.16.0.10', 'end': '172.16.0.200'}]
+  TenantAllocationPools: [{'start': '172.17.0.10', 'end': '172.17.0.200'}]
+  StorageAllocationPools: [{'start': '172.18.0.10', 'end': '172.18.0.200'}]
+  StorageMgmtAllocationPools: [{'start': '172.19.0.10', 'end': '172.19.0.200'}]
+  ManagementAllocationPools: [{'start': '172.20.0.10', 'end': '172.20.0.200'}]
+  ExternalAllocationPools: [{'start': '172.21.0.10', 'end': '172.21.0.100'}]
+  # Set to the router gateway on the external network
+  ExternalInterfaceDefaultRoute: 172.21.0.1
+  PublicVirtualFixedIPs: [{'ip_address':'172.21.0.10'}]
```

Comment:
We are going to default to a 172.21.0.0/22 cidr, which gives us a range of 1022 ip addresses 172.21.0.0-172.21.3.255. Should this value be 172.21.0.2 (leaving 1 for the gateway)? http://www.subnet-calculator.com/cidr.php